docs/rancher/cloud-provider.md

```diff
@@ -184,4 +184,16 @@ Alternatively, you can specify the parameters by adding annotations to the servi
 | `cloudprovider.harvesterhci.io/healthcheck-failure-threshold` | string | false | Specifies the health check failure threshold. The default value is 3. The backend server will stop forwarding traffic if the number of health check failures reaches the threshold.
 | `cloudprovider.harvesterhci.io/healthcheck-periodseconds` | string | false |  Specifies the health check period. The default value is 5 seconds.
 | `cloudprovider.harvesterhci.io/healthcheck-timeoutseconds` | string | false | Specifies the timeout of every health check. The default value is 3 seconds.
-    
\ No newline at end of file
+
+ ### Sync Harvester Node's Topology to Guest Cluster Node   
+After configuring the topology of the Harvester node, the cluster node created by RKE2 will sync the topology labels of the Harvester node to the Guest Cluster automatically via CCM, and the topology labels of each Guest Cluster node will be the same as the host Harvester node.
+
+*NOTE:* Currently, the labels can only be synced when the node is first initialized. If the host node is changed, the guest cluster node needs to be changed or redeployed manually.
```

Comment:
can user re-sync these labels by restarting guest node vms?

Reply:
CCM only supports initialization when after created, Determined by taint `node.cloudprovider.kubernetes.io/uninitialized`.

related: https://github.com/kubernetes/cloud-provider/blob/a198976dd5110d909a1a0be8a058d46ad45588d6/controllers/node/node_controller.go#L383-L388

Reply:
It's better to let user specify scheduling rules during creation to avoid zone changes caused by vm restart

Reply:
updated